src/Passwordless/PasswordlessClient.cs

```diff
@@ -112,7 +112,7 @@ public PasswordlessClient(PasswordlessOptions options)
     public async Task<GetEventLogResponse> GetEventLogAsync(GetEventLogRequest request, CancellationToken cancellationToken = default) =>
         (await _http.GetFromJsonAsync($"events?pageNumber={request.PageNumber}&numberOfResults={request.NumberOfResults}",
             PasswordlessSerializerContext.Default.GetEventLogResponse,
-            cancellationToken)) ?? new GetEventLogResponse();
+            cancellationToken))!;
```

Comment:
I did this because I assume the endpoint never actually returns null, just like the rest of them